identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -61,7 +69,6 @@ <h2 class="form__heading">Options</h2>
                         </a>
                     </li>
                     <li>
-                        <p class="label-like">No longer want to receive any of these emails?</p>
```

Comment:
I wonder whether there is a subtle implication here that we might need to keep.  This message explicitly says *these* emails vs. the button which says *all* emails. Technically *these* is more accurate because we just unsubscribe them from emails on the page.

Distinction might not be important though.

Reply:
Changed to "unsubscribe from all newsletters"